versions.tf

```
@@ -1,4 +1,11 @@
 
 terraform {
   required_version = ">= 0.12"
+
+  required_providers {
+    aws = {
+      version = ">= 4.0"
```

Do you want to constrain this to non-major changes, to avoid unexpectedly getting a breaking change?
```suggestion
      version = ">= 4.0, < 5"
```